src/main/resources/the_warlordResources/localization/eng/WarlordMod-Card-Strings.json

```diff
@@ -159,9 +158,9 @@
   },
   "the_warlord:Ambush": {
     "NAME": "Ambush",
-    "DESCRIPTION": "If this is the first card played this turn, deal !D! damage to ALL enemies. NL Exhaust.",
+    "DESCRIPTION": "Deal !D! damage. NL If the enemy is not attacking NL Gain [E]",
```

Comment:
'Gain' shouldn't be capitalized. And the NL is only needed if it makes the spacing look better, but that's up to you.